relayer/core/relay.go

```diff
@@ -22,13 +22,16 @@ import (
 	"github.com/snowfork/polkadot-ethereum/relayer/chain/parachain"
 	"github.com/snowfork/polkadot-ethereum/relayer/chain/relaychain"
 	"github.com/snowfork/polkadot-ethereum/relayer/store"
+	"github.com/snowfork/polkadot-ethereum/relayer/workers/parachaincommitmentrelayer"
```

Comment:
Nit: The package name `parachaincommitmentrelayer` is a bit too long, and `commitment` is just an internal implementation detail. So can shorten the name to `parachainrelayer`.